backend/core/src/listings/tests/listings.service.spec.ts

```diff
@@ -392,4 +392,16 @@ describe("ListingsService", () => {
       )
     })
   })
+
+  describe("getListingsExternal", () => {
+    it("should return the same as /listings", async() => {
```

Comment:
To make this test better test what it claims to, I would instead check that `service.listingsIncludeExternal()` is identical to `service.listings()`